layouts/partials/tw/form-contact.html

```diff
@@ -0,0 +1,203 @@
+{{ $subjects := slice
```

Comment:
Make shortcodes/featured/promo-contact-form.html call into this partial so there's no duplication.